app/components/autocomplete/at_mention_item/at_mention_item.tsx

```diff
@@ -50,24 +52,23 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme: Theme) => {
         },
         rowInfo: {
             flexDirection: 'row',
-            flex: 1,
```

Comment:
Can you make the shared channel Icon visible in code and fix this?  
<img width="378" alt="Screenshot 2021-07-09 at 11 07 55" src="https://user-images.githubusercontent.com/3680799/125028599-f3d54c80-e0a5-11eb-89fd-d507aebcc16d.png">